recipe/meta.yaml

```diff
@@ -621,6 +625,9 @@ outputs:
       files:
         - snippets/
       commands:
+        # https://conda-forge.org/docs/maintainer/knowledge_base/#newer-c-features-with-old-sdk
+        # Address: error: 'path' is unavailable: introduced in macOS 10.15
+        - export CXXFLAGS="${CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY"    # [osx and x86_64]
```

Comment:
why not to set it by default in macOS conda env? why every recipe should care about it?

Reply:
That’s a good question. We should discuss it in the ctng activation feedstock

Reply:
https://github.com/conda-forge/ctng-compiler-activation-feedstock/issues/144